### DISCUSSION TITLE
Unconditional MAISI sampler

### DISCUSSION THREAD

**Discussion by Z7Gao** [Project-MONAI/MONAI#7952] [General]
Thanks to the MAISI's team for creating this awesome foundation model. I have been playing with the ControlNet version and it yields pretty nice results.

Since the news on the Nvidia webiste involves unconditional sampling results, I wonder if that part of the code can also be released.

**Comment by KumoLiu** [comment]
Hi @Z7Gao, I'm not sure which page you mean.
The MAISI is not completely merged into the core, the tutorial is still WIP.
I guess you can get more information here: https://github.com/Project-MONAI/tutorials/tree/main/generative

Convert it to discussion, since not an issue.
Thanks.

**Reply by Can-Zhao** [reply]
It is ready now. [MAISI](https://github.com/Project-MONAI/tutorials/tree/main/generation/maisi) ([https://github.com/Project-MONAI/tutorials/tree/main/generation/maisi](https://github.com/Project-MONAI/tutorials/tree/main/generation/maisi))
